docs/BOUNTIES.md

```diff
@@ -2,6 +2,8 @@
 
 Get paid to improve openpilot!
 
+## Please check [here](https://github.com/orgs/commaai/projects/26/views/1) for a full list of open bounties
```

Comment:
does this have the bounties from the other repos yet? there's a link down below that has all of them

Reply:
added

Reply:
cool, let's replace that search with this then